helm.yml

```diff
@@ -12,15 +12,19 @@ plugins:
   - flant-statusmap-panel
   - simpod-json-datasource
   - grafana-piechart-panel
+datasources:
```

Comment:
Will we be removing this in a later commit?  Is it worth considering (as a later bit of work) using this approach to set all datasources to editable and then removing from the helm config?

Reply:
Yes mate I'll raise another PR to remove it once sorted in prod as otherwise it just adds in an empty elasticsearch data source when you re-run the pipeline. I can understand why some of the forum posts on Grafana state removing these are so difficult 😢 

For Elasticsearch data source though I created a new data source whereas the others like Azure Monitor we wouldn't need to so perhaps a hot fix branch would be better for them? Deploy the hot fix branch to make editable and then remove the branch so it doesn't happen again.